app/src/main/res/values/style.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="MainTitleStyle">
+        <item name="android:textColor">@color/white</item>
+        <item name="android:textSize">@dimen/dimen_text_main_big</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="ButtonMainStyle">
+        <item name="android:backgroundTint">@color/yellow</item>
```

Comment:
a style for only one attribute is not a good practice, check if you can add more attributes from the view or delete this style pls

Reply:
done, I put the color in activity_main(button) and deleted this style